checkpoint/requirements.txt

```diff
@@ -0,0 +1,10 @@
+gensim==3.6
```

Comment:
hopefully (assuming we can get rid of the custom COMPARE-CHECKPOINTS docker image), then we won't need this requirements? or let's discuss